pages/create-collection.tsx

```diff
@@ -0,0 +1,17 @@
+import type { NextPage } from "next";
+import Head from "next/head";
+
+const createCollection: NextPage = function () {
+  return (
+    <div>
+      <Head>
+        <title>Create Next App</title>
```

Comment:
Fix the head content please.
"Hedera Minting" instead of "Create Next App", same for the content attribute.